File: Core/Stateflows.Common/Events/InitializationRequest.cs
using System.Collections.Generic;

namespace Stateflows.Common
{
    public class InitializationRequest<TResponse> : Request<TResponse>
        where TResponse : InitializationResponse
    {
        public override string Name => nameof(InitializationRequest);

        public Dictionary<string, object> Values { get; set; } = new Dictionary<string, object>();
    }

    public class InitializationRequest : InitializationRequest<InitializationResponse>
    { }
}

File: Transport/@stateflows/common/src/events/initialization.request.ts
import { Request } from "./request";
import { InitializationResponse } from "./initialization.response";

export class InitializationRequest extends Request<InitializationResponse> {
    public $type = "Stateflows.Common.InitializationRequest, Stateflows.Common";
    public Name = "Stateflows.Common.InitializationRequest";
}